### PR TITLE
Organ pages good-enough and linked from Atlas menu

### LINKED ISSUE #1824: Design Organ Detail Pages
Todo:

- ID info that needs to be displayed on a tissue-specific page (organ = organ type, ie all kidneys, not specific sample)
- Description text
- Possible links: Azimuth datasets,  reference object used by CCF, ASCT+B table, for all organs of that type a built-in faceted browsing interface for that datatype
- List of TMCs generating data on this organ
- Overview of assays

### LINKED ISSUE #2090: Implement Organ pages
[diagram](https://docs.google.com/presentation/d/1ZznSezXx-vnIlTRMafdMquFWlsSsrtcbV-qCQK7SNZ4/edit#slide=id.p)

### Design:
- [x] #1824
- [x] #2091
- [x] #2092

### Implementation, details:
- [ ] Get organ descriptions
- [ ] Anatomy:
  - [ ] Get UBERON list with descriptions from IU
  - [ ] Get web component from IU
  - [ ] Add ES facets to description mapping from IU
- [ ] Reference-based analysis:
  - [x] #2126 
  - [ ] Get list of modalities for each organ
  - [ ] Get count of nuclei
  - [ ] Get list of reference datasets
  - [ ] Get link to Azimuth
  - [ ] Get Vitessce config
- [ ] Assays:
  - [ ] Query the assays
  - [ ] Make a barchart
- [ ] Samples:
  - [ ] Query the samples  

### Implementation, list:
- [ ] Implement Organ list

### Implementation, navigation from home:
- [ ] Implement link from home page

### LINKED ISSUE #2092: Design navigation from home page to Organ list

### LINKED ISSUE #2126: Pull in azimuth data, based on Uberon ID
Follow up from https://github.com/hubmapconsortium/portal-ui/issues/2107#issuecomment-912772003

[Diagram](https://docs.google.com/presentation/d/1ZznSezXx-vnIlTRMafdMquFWlsSsrtcbV-qCQK7SNZ4/edit#slide=id.p)

- [ ] [`azimuth_references.yaml`](https://github.com/satijalab/azimuth_website/blob/master/data/azimuth_references.yaml) needs to include UBERON ID for each. (Alternatively, change from a list to a dict keyed by UBERON ID.)
- [ ] [Vitessce configs](https://github.com/satijalab/azimuth_website/tree/master/assets/json) need to be refererenceable by UBERON ID. (The names in `azimuth_references.yaml` aren't sufficient to link to the corresponding vitessce configs: For example, "motor cortex" is not handled consistently.) Perhaps put all the vitessce configs in one large dict, keyed by UBERON ID? Implementation decision.
- [ ] When both of these are done, fix the script on our side to pull in the information.

### LINKED ISSUE #2177: link to organs from atlas menu

### PR DESCRIPTION
**Pull request by mccalluc**
cc @ngehlenborg / @tsliaw 

For the work I know still needs to be done, I've filed 
- #2182

Suggest that that be reviewed in conjunction with this, and if anything is absolutely essential, bring it forward.

Note that this also adds a util function that excludes entities that are not the latest version. We'll be adding a clause to this soon to also excludes retracted entities. 

Note that this adds "Organs" to the "Atlas & Tools" Menu. If we want to have this up on Friday, it needs to be in this release.

- Fix #2177 
- Fix #2126
- Fix #2092
- Fix #2090
- Fix #1824

### CODE REVIEW

**Review by john-conroy — COMMENTED** [review]
I'm pretty sure `useMemo` is needed here, but I can confirm later today when I get a chance to check it out locally.

File: context/app/static/js/components/organ/Assays/Assays.jsx [hunk 1]
```diff
@@ -1,31 +1,92 @@
...
 import { SpacedSectionButtonRow } from 'js/shared-styles/sections/SectionButtonRow';
+import useSearchData from 'js/hooks/useSearchData';
 
 import { getSearchURL } from '../utils';
 
 function Assays(props) {
   const { searchTerms } = props;
   const searchUrl = getSearchURL('Dataset', searchTerms);
 
+  const query = {
```
[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/organ/Assays/Assays.jsx:22]: You'll know if it sends extra requests.

[the remark this answers is not in the document]
Comment (mccalluc) [reply context/app/static/js/components/organ/Assays/Assays.jsx:22]: ... and it certainly is. Thanks for catching!

File: context/app/static/js/components/organ/Azimuth/Azimuth.jsx [hunk 2]
```diff
@@ -12,25 +12,25 @@ import { StyledPaper } from './style';
...
         {/* eslint-disable react/no-danger */}
         <LabelledSectionText label="Reference dataset">
-          <div dangerouslySetInnerHTML={{ __html: dataRefHtml }} />
+          <span dangerouslySetInnerHTML={{ __html: dataRefHtml }} />
         </LabelledSectionText>
       </StyledPaper>
-      TODO: Refactor so that can get the spinner without the a title?
+
+      {/* Need to refactor so "Visualization" is not included. */}
+      <br />
```
[the remark this answers is not in the document]
Comment (mccalluc) [reply context/app/static/js/components/organ/Azimuth/Azimuth.jsx:33]: I'll add that to the TODO: The component below needs to be reworked extensively: Without a bit of space the Vitessce controls run into the paper above.

File: context/app/static/js/helpers/functions.js [hunk 3]
```diff
@@ -69,3 +71,7 @@ export function tableToDelimitedString(rows, colNames, d) {
 export function createDownloadUrl(fileStr, fileType) {
   return window.URL.createObjectURL(new Blob([fileStr], { type: fileType }));
 }
+
+export function getDefaultQuery() {
+  return BoolMustNot(ExistsQuery('next_revision_uuid'));
+}
```
[the remark this answers is not in the document]
Comment (mccalluc) [reply context/app/static/js/helpers/functions.js:77]: Sure, I'll change this to plain JSON. Is the motivation avoiding any more lock-in to that library?

**Comment by mccalluc** [comment]
Interpreting "comment" as "ok to merge with changes"...

**Pull request merged**